### DISCUSSION TITLE
Running Zed as a Docker Container

### DISCUSSION THREAD

**Discussion by MyCodeBits** [zed-industries/zed#17189] [Feature Requests]
My requirement to have Zed IDE open in the browser. 
- One of the option is that if Zed project is available as a Docker image. Is there a known way to run Zed IDE as a Docker container? 
- Any other option by which it can be exposed on a port and accessible as locahost:[PORT] ?
Thanks.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
